source/release-notes.txt

```diff
@@ -10,6 +10,19 @@ Release Notes
    :depth: 1
    :class: twocols
 
+|compass| 1.45.0
+----------------
+
+*Released December 4, 2024*
+
+Bug Fixes:
+
+- Disables UTF8 validation when retrieving Compass metadata 
+  (:issue:`COMPASS08517`)
```

Comment:
[typo]

I think this ought to be `COMPASS-8517`.

Reply:
Ack sorry about missing that 😓 Updating.